tools/mtf

```diff
@@ -13,8 +15,39 @@ positional arguments:
   AVOCADO_ARGS        test files and any arguments that "avocado run" accepts
 
 optional arguments:
+  -a, --action        run selected avocado command (DEFAULT: run)
   -h, --help          show this help message and exit
   -l, --linters       also run default linter tests included in MTF
+  -t, --tags T1,-T2   set test filtering based on tags, use --taglist for help ()
```

Comment:
remote (-) before T2

Reply:
It is intended there. It as example that you can use __negation__ of filter  meaning is T1 AND (NOT T2) it is semantic provided by avocado.